integration-tests/test-common/src/test/java/org/apache/servicecomb/tests/tracing/TracingTestBase.java

```diff
@@ -62,7 +62,11 @@ protected void assertThatSpansReceivedByZipkin(Collection<String> logs, String..
         .map(this::extractIds)
         .collect(Collectors.toList());
 
-    List<Span> spans = zipkin.getTrace(traceId(loggedIds));
+    List<Span> spans;
+    do {
+      spans = zipkin.getTrace(traceId(loggedIds));
```

Comment:
we need to sleep in case of cpu 100%, also we need a reasonable timeout.
Howerver, instead of write our impl, we can use https://github.com/awaitility/awaitility library
```
Awaitility.await().atMost(10, SECONDS).until(() -> xxxxxx);
```

Reply:
just testing code, that almost never happens, don't need to think so much.   of course, it's better to be strict !